exer2-http-airport-json-parse.py

```diff
@@ -30,6 +29,14 @@
             else:
                 lrg = "-"
 
-            print("{0:10} {1:30} {2:20} {3:20}".format(id, name, avg, lrg))
+            sanitized_data.append((airline_id, name, avg, lrg))
+
+# Display Data
 
+template = "{0:10} {1:30} {2:20} {3:20}"
+print("=" * 10, "=" * 30, "=" * 20, "=" * 20)
```

Comment:
strings can use * operator.  This makes it a bit easier to align with columns from your template, and makes the code a bit cleaner IMO